data/grass.json

```diff
@@ -11,63 +11,151 @@
+                    "code": "sudo port install grass",
+                    "url": "https://ports.macports.org/port/grass/"
+                }
             }
         },
         "linux":{
-            "arch_linux": {
-                "id": "arch_linux",
-                "title": "Arch Linux",
-                "code": "sudo pacman -S grass",
-                "url": "https://aur.archlinux.org/packages/grass"
-            },
-            "debian": {
-                "id": "debian",
-                "title": "Debian",
-                "code": "sudo apt-get install grass",
-                "url": "https://packages.debian.org/search?keywords=grass"
-            },
-            "generic_64_bit":{
-                "id": "generic_64_bit",
-                "title": "Generic 64-bit",
-                "code": "add me",
-                "url": "https://grass.osgeo.org/grass84/binary/linux/snapshot/"
-            },
-            "EPEL": {
-                "id": "EPEL",
-                "title": "EPEL",
-                "code": "sudo yum install grass",
-                "url": "https://src.fedoraproject.org/rpms/grass"
-            },
-            "fedora":{
-                "id": "fedora",
-                "title": "Fedora",
-                "code": "sudo dnf install grass",
-                "url": "https://apps.fedoraproject.org/packages/grass"
-            },
-            "gentoo": {
-                "id": "gentoo",
-                "title": "Gentoo",
-                "code": "sudo emerge grass",
-                "url": "https://packages.gentoo.org/packages/sci-geosciences/grass"
-            },
-            "openSUSE": {
-                "id": "openSUSE",
-                "title": "openSUSE",
-                "code": "sudo zypper install grass",
-                "url": "https://build.opensuse.org/package/show/Application:Geo/grass"
-            },
-            "ubuntu": {
-                "id": "ubuntu",
-                "title": "Ubuntu",
-                "code": "sudo apt-get install grass",
-                "url": "https://launchpad.net/~ubuntugis/+archive/ubuntu/ubuntugis-unstable"
+            "binaries" : [
+                {
+                    "id": "source-8.4.1",
+                    "title": "Current - 8.4.1 (tar.gz)",
+                    "size": "66.7 MB",
+                    "date": "2025-02-24",
+                    "md5": "https://download.osgeo.org/grass/grass84/source/grass-8.4.1.md5sum",
+                    "url": "https://download.osgeo.org/grass/grass84/source/grass-8.4.1.tar.gz"
+                },
+                {
+                    "id": "source-8.4.0",
+                    "title": "Current - 8.4.0 (tar.gz)",
+                    "size": "67.0 MB",
+                    "date": "2024-07-27",
+                    "md5": "https://download.osgeo.org/grass/grass84/source/grass-8.4.0.md5sum",
+                    "url": "https://download.osgeo.org/grass/grass84/source/grass-8.4.0.tar.gz"
+                }
+            ],
```

Comment:
These are not binaries, so why to have them here? I think we don't really have anything downloadable, so maybe we should remove the button.

Reply:
Yes, I forgot to remove this when I removed the Download button from this section.